builder/test/softmax.cc

```diff
@@ -139,9 +145,19 @@ TEST_P(softmax, pipeline) {
       {{exp_in, {c, b}}, {sum_exp_in, {b}}}, call_stmt::attributes{.name = "exp_in"});
   func pass3 = func::make(normalize, {{exp_in, {all_c, point(b)}}, {sum_exp_in, {point(b)}}}, {{softmax_out, {c, b}}},
       call_stmt::attributes{.name = "normalize"});
-  // Add a trivial consumer so we can have an inner loop here too.
-  func pass4 = func::make(
-      add_1<float>, {{softmax_out, {point(c), point(b)}}}, {{out, {c, b}}}, call_stmt::attributes{.name = "consumer"});
+
+  func pass4, copy;
```

Comment:
Can this be reproduced by just adding loops around pass3 and not pass4, instead of adding a new stage?

Reply:
Yes, but the interesting thing about copy is that it will be simplified away in the final IR.

Reply:
Done.